### PR TITLE
Keep the source code, update the README

### PR DESCRIPTION
**Pull request by mmagician**
Open Grants companion: https://github.com/w3f/Open-Grants-Program/pull/493

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
We also need to update https://github.com/w3f/General-Grants-Program/blob/master/grants/grants.md If I remember correctly some links previously pointed to this page. So better safe than sorry.

File: README.md [hunk 1]
```diff
@@ -1,175 +1,4 @@
-# General Grants Program <!-- omit in toc -->
+# General Grants Program
 
-<p align="center">
-  <img src="src/General_Grants_Program.png" style="width:1300px";>
-</p>
+This repository has been deprecated in favour of the [Open Grants repository](https://github.com/w3f/Open-Grants-Program/). You can still apply for a General Grant - to do so, follow the steps outlined in Open Grant Program's README.
```
Comment (alxs) [inline README.md:3]: Perhaps we could even remove this distinction? We'd simply have a W3F Grants Program and for applications that that are private, above 100k and all that - a link to the form. I think David initially suggested this.

**Comment by semuelle** [comment]
> We also need to update [`master`/grants/grants.md](https://github.com/w3f/General-Grants-Program/blob/master/grants/grants.md) If I remember correctly some links previously pointed to this page. So better safe than sorry.

Can we remove the file or just replace its contents with `Moved to ./README.md`? I don't think we should continue updating two files.

**Comment by mmagician** [comment]
> We also need to update https://github.com/w3f/General-Grants-Program/blob/master/grants/grants.md If I remember correctly some links previously pointed to this page. So better safe than sorry.

@Noc2 done

**Comment by mmagician** [comment]
> Can we remove the file or just replace its contents with `Moved to ./README.md`? I don't think we should continue updating two files.

@semuelle That's hopefully the last update this file will ever see :)

File: README.md (hunk 1 above)
Comment (semuelle) [reply README.md:3]: I don't strictly favor one over the other, but I think it'd help to have a separate name for it, just so we can point people to the right resources.

**Comment by semuelle** [comment]
> @semuelle That's hopefully the last update this file will ever see :)

You're right, nevermind.

**Review by semuelle — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Noc2) [reply README.md:3]: I would be in favor of just calling it Web3 Grants.

Comment (Noc2) [reply README.md:3]: Maybe we could add an explanation here that now it’s only Web3 Grants and that we merged the old open grants + general grants into one repo.

Comment (alxs) [reply README.md:3]:
```suggestion
This repository has been deprecated in favour of the [W3F Grants repository](https://github.com/w3f/Grants-Program/).
```

**Pull request merged**